core/include/kernel_init.h

```diff
@@ -25,6 +25,18 @@
 extern "C" {
 #endif
 
+#ifdef DOXYGEN
+/**
```

Comment:
These can be grouped and added to the `config` Doxygen group.

Reply:
Ok, grouped them.